build.gradle

```diff
@@ -50,6 +50,9 @@ repositories {
 		name = "Modmuss50"
 		url = "http://maven.modmuss50.me/"
 	}
+	flatDir {
```

Comment:
Why this? I do not see the railcraft jar, and I would be a lot happier using maven, even if it was from curse's maven endpoint.

Reply:
railcraft does not have a proper maven set up now unfortunately. i might see if railcraft can use bintray or something similar.